pkgs/applications/editors/vim/plugins/deprecated.json

```diff
@@ -7,6 +7,10 @@
         "date": "2021-12-21",
         "new": "cmp-tmux"
     },
+    "taskwarrior": {
+        "date": "2024-08-13",
+        "new": "taskwarrior3 or taskwarrior3"
```

Comment:
Is this supposed to be "taskwarrior2 or taskwarrior3"?

Reply:
> Is this supposed to be "taskwarrior2 or taskwarrior3"?

Yes it is. Thanks for catching that!